### PR TITLE
fix: Work by default with webpack

### LINKED ISSUE #10: Add typescript/webpack configuration to readme
I am creating a new issue for #7 as the original owner closed it despite the issue not being resolved. If you are having issues getting cypress-axe to work with Typescript/Webpack, please look at that issue for solutions.

### PR DESCRIPTION
**Pull request by NicholasBoll**
Fixes #10

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Cypress.Commands.add('injectAxe', () => {
   cy.window({ log: false }).then(window => {
     const axe = require('axe-core')
-    window.eval(axe.source)
```
Comment (NicholasBoll) [inline index.js]: Is there a reason the original code ran `window.eval(axe.source)` instead of `window.axe = axe`?

File: index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+    window.axe = axe
```
Comment (NicholasBoll) [inline index.js]: We already have access to the `axe` reference, just assign it to the the app's `window` object so it is available to other `cypress-axe` commands. They refer to `axe` via the `window` object already.

**Comment by Capocaccia** [comment]
@avanslaars Please update on this PR when you have a chance.  We are facing this same issue and validated that the fix submitted by @NicholasBoll works.

**Comment by avanslaars** [comment]
@all-contributors please add @NicholasBoll for code

**Comment by Capocaccia** [comment]
Thank you @avanslaars  @NicholasBoll ! Your work on this project is highly appreciated!

**Comment by wKovacs64** [comment]
This change broke every single project I have that uses `cypress-axe`. 😅 FWIW, I'm using the [Browserify preprocessor](https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/preprocessors__typescript-browserify), not Webpack. Assertions fail, flagging false positives, etc.

**Comment by avanslaars** [comment]
I'm reverting this change for now. I should have done some testing to verify this didn't break existing behavior. Since Browserify is the default that ships with Cypress, this change is going to break more projects than it fixes.

I'm happy to make this work for multiple pre-processors, but it'll have to be done in a way that doesn't break the others.

**Comment by avanslaars** [comment]
@wKovacs64 That fix is out there.

@Capocaccia @NicholasBoll - we'll have to find another way to fix the issues when using webpack. Can one of you point me to a small example repo with the issue?

**Comment by wKovacs64** [comment]
Thanks @avanslaars

**Pull request merged**